protos/Robot_Echo/SRRobot.proto

```diff
@@ -14,980 +14,979 @@ PROTO SRRobot [
-                  ]
-                  creaseAngle 1
+                geometry DEF GEO_DS Box {
+                  size 0.03 0.03 0.03
                 }
               }
             ]
-            name "right wheel"
-            boundingObject DEF BOUNDING_WHEEL Group {
-              children [
-                Transform {
-                  rotation 0 0 1 1.5708
-                  children [
-                    Cylinder {
-                      height 0.01
-                      radius 0.1
-                      subdivision 32
-                    }
-                  ]
-                }
-              ]
-            }
-            physics Physics {
-              density 20000
-            }
+            name "Back Left DS"
+            lookupTable [
+              0 0 0
+              2 1000 0
+            ]
           }
-        }
-        Solid {
-          translation 0.19999998777297837 0.00026863610916181684 -0.00036030151878293246
-          rotation -0.9999999953968224 4.284946735166644e-05 8.585032564057752e-05 0.9344151591877564
-          boundingObject DEF BOUNDING_WHEEL Group {
+          DEF front_right DistanceSensor {
+            translation 0.12 0.17 -0.01
```

Comment:
I'm not sure we should be moving the distance sensors or the joints or any physical attributes of the robot which might impact its behaviour.